lte/protos/mobilityd.proto

```diff
@@ -24,8 +24,8 @@ option go_package = "magma/lte/cloud/go/protos";
 // --------------------------------------------------------------------------
 message IPAddress {
   enum IPVersion {
-    IPV4 = 0;
-    IPV6 = 1;
+    IPv4 = 0;
```

Comment:
Do we really need this change? If yes can you go through the codebase and replace `IPAddress_IPV4` with `IPAddress_IPv4`